src/arrayReverse.test.js

```diff
@@ -7,14 +7,48 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should handle an empty array`, () => {
+    const result = arrayReverse([]);
 
+    expect(result).toEqual([]);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it('should reverse a single string in the array', () => {
+    const result = arrayReverse(['Hell0']);
+
+    expect(Array.isArray(result)).toBe(true);
+    expect(result).toEqual(['0lleH']);
+  });
+
+  it('should reverse multiple strings in the array', () => {
+    const result = arrayReverse(['Mate', 'Academy']);
+
+    expect(result).toEqual(['ymed', 'acAetaM']);
   });
 
-  // write more tests here
+  it('should reverse strings with varying lengths and keep the order', () => {
+    const result = arrayReverse(['I', 'am', 'a', 'student!']);
+
+    expect(result).toEqual(['!', 'tn', 'e', 'dutsamaI']);
+  });
+
+  it('should handle strings with special characters and numbers', () => {
```

Comment:
This can be split into two tests: one for numbers, one for special characters

Reply:
Thank you!